### PR TITLE
43 search by dates

### PR DESCRIPTION
**Pull request by junyuanxue**
connect #43

### CODE REVIEW

**Comment by junyuanxue** [comment]
connects #47

File: app/controllers/spaces.rb [hunk 1]
```diff
@@ -14,12 +14,25 @@ class MakersBnb < Sinatra::Base
 
   get '/space/index' do
     @spaces = Space.all
+    @searched_dates = session[:searched_dates]
+		p "there"
```
Comment (rhiannonruth) [inline app/controllers/spaces.rb]: delete p

File: app/controllers/spaces.rb [hunk 2]
```diff
@@ -14,12 +14,25 @@ class MakersBnb < Sinatra::Base
...
     erb(:"space/index")
   end
 
+  post '/space/search' do
+    session[:searched_dates] = get_date_range(js_to_rb_date(params[:start_date]), js_to_rb_date(params[:end_date]))
+		p "here"
```
Comment (rhiannonruth) [inline app/controllers/spaces.rb]: delete line

**Comment by rhiannonruth** [comment]
:shipit:

**Pull request merged**